lib/rgb_matrix/engine.ex

```diff
@@ -27,10 +27,10 @@ defmodule RGBMatrix.Engine do
   - `initial_animation` - The Animation type to initialize and play when the
     engine starts.
   """
-  @spec start_link({leds :: [LED.t()], initial_animation_type :: Animation.type()}) ::
+  @spec start_link(leds :: [LED.t()]) ::
           GenServer.on_start()
```

Comment:
:information_source: (optional)

Does the return type fit on the same line now?